MekHQ/src/mekhq/campaign/mission/Scenario.java

```diff
@@ -1023,6 +1011,8 @@ public static Scenario generateInstanceFromXML(Node wn, Campaign c, Version vers
 
                 if (wn2.getNodeName().equalsIgnoreCase("name")) {
                     retVal.setName(wn2.getTextContent());
+                } else if (wn2.getNodeName().equalsIgnoreCase("scenarioType")) {
+                    retVal.setStratConScenarioType(ScenarioType.fromOrdinal(Integer.parseInt(wn2.getTextContent())));
```

Comment:
purely stylistic choice, I prefer to use the enum name instead of the ordinal value, as there is no meaningful change here and allow us to add more values in betwen enums, and is human readable.

Reply:
Yeah, I think that would be best, too. Originally I didn't use enum name because I couldn't work out a way to handle that without some really clunky code. Reading through your review popped the answer in my head, so I'll get that done.

Reply:
Enum.valueOf is a default in Java and transform the `name` into the Enum.